README.md

```diff
@@ -95,7 +89,8 @@ After you're finished please remove all the comments and instructions!
 
 ### Key Features <a name="key-features"></a>
 
-> Describe between 1-3 key features of the application.
+- **[html]**
+- **[css]**
 
 - **colored header**
 
```

Comment:
- [ ] Kindly implement the changes for Readme as previous reviewer suggested.

![Screenshot from 2022-12-04 01-31-26](https://user-images.githubusercontent.com/60083437/205458566-d5a6be7b-dc11-434b-85c2-e616fb869a6c.png)